src/ptvsd/wrapper.py

```diff
@@ -1739,12 +1726,13 @@ def version_str(v):
         self.send_response(request, **sys_info)
 
     # VS specific custom message handlers
-    @async_handler
+
     def on_setDebuggerProperty(self, request, args):
         if 'JustMyCodeStepping' in args:
             jmc = int(args.get('JustMyCodeStepping', 0)) > 0
             self.debug_options['DEBUG_STDLIB'] = not jmc
 
+        self._forward_request_to_pydevd(request, args)
```

Comment:
`_forward_request_to_pydevd` already sends a response, so, it should no longer send the response in this method.